### DISCUSSION TITLE
Email has already been used to obtain an academic discount on GitHub

### DISCUSSION THREAD

**Discussion by EEKHS** [community/community#55950] [GitHub Education]
### Hi everyone,

I managed to delete the account I received the teacher benefits on. I made a new account under my same school email address, but when I try to verify, I am getting an "email has already been used to obtain an academic discount" error. How can I change the account that is associated with the teacher program from my deleted one to the new one?

**Comment by ryanlua** [comment]
Hi @EEKHS,

Can you try to just add that academic email to your GitHub account normally? If this doesn't work then you should [Get help with GitHub Education](https://support.github.com/contact/education) and ask them to remove your academic email from the deleted account.

**Comment by samus-aran** [comment]
Hi @EEKHS  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

In this particular case you will need to reach out to GitHub support [(Account & Billing)](https://support.github.com/features/account) to make adjustments.

**Reply by Dev-duarte** [reply]
Me pasa que perdí acceso a mi cuenta y desvinculé el correo educativo a mi vieja cuenta y cree una nueva con esta pero al momento de solicitar el Education pack me dice que ya lo está usando mi cuenta anterior

**Comment by artysan-code** [comment]
I have the same problem and seems to be a really bad bug with the data stored by those server, cuz if the server rejects the update of the University Data one time, you cannot access it anymore.

**Comment by hbahrami11** [comment]
Hbahrami

**Comment by Souzaarj-dev** [comment]
I'm experiencing the same problem, without being able to use the student benefit, even adding the academic email to a new account, after losing the mfa on the old account. It seems to me to be a convenient problem, since there will be one less student using the benefit.

![image](https://github.com/user-attachments/assets/728cc175-a27b-49ee-9eb8-914909683434)

**Comment by artysan-code** [comment]
I can say there's no solution, Github answered me in 30 days saying that I cannot do nothing cuz I deleted my previous account.

**Reply by SSea-man** [reply]
did they solved that problem? i felt regret now!
